ocs_ci/deployment/acm.py

```diff
@@ -191,18 +192,27 @@ def download_downstream_binary(self):
         )
         run_cmd(cmd)
         # Wait till image extract happens and subctl dir appears
-        time.sleep(30)
+        time.sleep(90)
+        run_cmd("sudo chmod 777 /tmp/subctl*", shell=True)
```

Comment:
hmm, why we need to do it with sudo? I am not sure it's a good idea to require to have sudo here for framework. Do we really need 777? 

Shouldn't we rather use something like:
`os.chmod(file_path, 0o755)` ?

Reply:
removed this as well